src/Compilers/Core/Portable/SourceGeneration/RunResults.cs

```diff
@@ -142,6 +142,9 @@ public ImmutableArray<SyntaxTree> GeneratedTrees
         /// <summary>
         /// A collection of the named incremental steps executed during the generator pass this result represents.
         /// </summary>
+        /// <remarks>
+        /// Contains only final output steps unlike <see cref="TrackedSteps"/> which contains also intermediate steps.
```

Comment:
Does TrackedSteps only contain intermediate steps where `WithTrackingName` was called?

If that is the case, consider clarifying this part as well.

It would also be a good idea if this is distinguished in the summary